tools/extra/ras/main.c

```diff
@@ -909,14 +909,50 @@ fpga_result inject_ras_errors(fpga_handle fme_handle,
 	return result;
 }
 
+fpga_result clear_fpga_errors(fpga_handle handle, char *error )
```

Comment:
You might want to make this STATIC, and the error a `const char *` type
```
STATIC fpga_result clear_fpga_errors(fpga_handle handle, const char *error )
```

Reply:
fixed